cmd/oras/manifest/push.go

```diff
@@ -35,9 +38,11 @@ type pushOptions struct {
 	option.Pretty
 	option.Remote
 
-	targetRef string
-	fileRef   string
-	mediaType string
+	concurrency int64
+	targetRef   string
+	fileRef     string
+	mediaType   string
+	extraRefs   []string
```

Comment:
nit: it is better to put line 45 between line 42-43.

Reply:
done